backend/src/test/resolvers.spec.js

```diff
@@ -0,0 +1,590 @@
+                id
+              }
+            }
+          `;
+          let res = await mutate({ mutation: invalidValueMut });
+          expect(res.errors).toEqual([new Error("Invalid vote value.")]);
+        })
+
+        it("Value = 2", async () => {
+          const invalidValueMut = gql`
+            mutation {
+              vote (postId: "ckjokcgf42fm50a49osudvspq", voteValue: 2){
+                id
+              }
+            }
+          `;
+          let res = await mutate({ mutation: invalidValueMut });
+          expect(res.errors).toEqual([new Error("Invalid vote value.")]);
+        })
+
+        it("Value = 10", async () => {
+          const invalidValueMut = gql`
+            mutation {
+              vote (postId: "ckjokcgf42fm50a49osudvspq", voteValue: 10){
+                id
+              }
+            }
+          `;
+          let res = await mutate({ mutation: invalidValueMut });
+          expect(res.errors).toEqual([new Error("Invalid vote value.")]);
+        })
+
+        it("Value = -2", async () => {
+          const invalidValueMut = gql`
+            mutation {
+              vote (postId: "ckjokcgf42fm50a49osudvspq", voteValue: -2){
+                id
+              }
+            }
+          `;
+          let res = await mutate({ mutation: invalidValueMut });
+          expect(res.errors).toEqual([new Error("Invalid vote value.")]);
+        })
+
+        it("Value = -10", async () => {
+          const invalidValueMut = gql`
+            mutation {
+              vote (postId: "ckjokcgf42fm50a49osudvspq", voteValue: -10){
+                id
+              }
+            }
+          `;
+          let res = await mutate({ mutation: invalidValueMut });
+          expect(res.errors).toEqual([new Error("Invalid vote value.")]);
+        })
+      })
+    })
+  })
+
+  describe("Signup", () => {
+    it("Signs someone up successfully", async () => {
+      const signupMut = gql`
+        mutation {
+          signup(name: "Peter", email: "mail@mail.de", password: "password")
+        }
+      `;
+      let res = await mutate({ mutation: signupMut });
+      expect(res.data).toEqual({signup: expect.any(String)});
+    });
+
+    it("tries to register existing user", async () => {
+      const signupMut = gql`
+        mutation {
+          signup(
+            name: "Peter"
+            email: "günther.jauch@aol.de"
+            password: "password"
+          )
+        }
+      `;
+      let res = await mutate({ mutation: signupMut });
+      expect(res.data).toEqual({
+        signup: "Email already signed up.",
+      });
+    });
+
+    it("Provides invalid password error", async () => {
+      const signupMut = gql`
+        mutation {
+          signup(name: "Peter", email: "mail@mail.de", password: "pass")
+        }
+      `;
+      let res = await mutate({ mutation: signupMut });
+      expect(res.data).toEqual({
+        signup: "Password invalid. Choose another password.",
+      });
+    });
```

Comment:
In order to test if the remote schema would be called, you could inject `executor` into your schema from within your test cases. Your `executor` would be a mock function and you could ask:
```js
expect(executor).toHaveBeenCalled()
```
https://jestjs.io/docs/en/expect#tohavebeencalled